tools/mpremote/mpremote/main.py

```diff
@@ -16,7 +16,6 @@
     mpremote fs <command> <args...>  -- execute filesystem commands on the device
     mpremote repl                    -- enter REPL
 """
-
```

Comment:
Unless Black is doing this automatically, please don't change existing formatting/style.

Reply:
Indeed due to black. v22.3.0. `black -l99 <file>`